src/utils.ts

```diff
@@ -108,24 +107,24 @@ export function makeFormatter(options: Intl.DateTimeFormatOptions): () => Intl.D
   }
 }
 
-let dayFirst: boolean | null = null
+const dayFirst: Record<string, boolean> = {}
 const dayFirstFormatter = makeFormatter({day: 'numeric', month: 'short'})
 
 // Private: Determine if the day should be formatted before the month name in
 // the user's current locale. For example, `9 Jun` for en-GB and `Jun 9`
 // for en-US.
 //
 // Returns true if the day appears before the month.
-export function isDayFirst(): boolean {
-  if (dayFirst !== null) {
```

Comment:
@keithamus I only medium understand what is happening, but can we safely remove this check?

Reply:
`dayFirst` cannot be null as it is a const assigned to an object.